source/getting-started/use_cases/cloud_security.rst

```diff
@@ -99,8 +105,10 @@ Expand the output to see an example of an alert generated when an AWS security g
 
 Example of AWS dashboard:
 
-.. thumbnail:: ../../images/getting_started/use_case_cloud.png
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-cloud-geo.png
```

Comment:
This image needs to be edited. I can see the black line from the header you removed.

![image](https://user-images.githubusercontent.com/79285586/141319889-aaec0b43-5063-4770-8f2b-463b26f24483.png)